src/MetaMorpho.sol

```diff
@@ -748,9 +766,10 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         }
     }
 
-    /// @dev Fakes a withdraw of `assets` from the idle liquidity and Morpho if necessary.
+    /// @dev Simulates a withdraw of `assets` from the idle liquidity and Morpho if necessary.
+    /// @dev Warning: Can revert.
     /// @return remaining The assets left to be withdrawn.
-    function _staticWithdrawMorpho(uint256 assets) internal view returns (uint256 remaining) {
+    function _simulateWithdrawMorpho(uint256 assets) internal view returns (uint256 remaining) {
```

Comment:
Why is this change in this PR? Shouldn't it be based on #280 ?

Reply:
I'll pull the `cantina-review` if you want

Reply:
Actually there are some conflicts so it's mandatory anyway